test/pytest/test_hotplug.py

```diff
@@ -0,0 +1,18 @@
+import serial
+import pytest
+from pexpect.fdpexpect import fdspawn
+
+@pytest.mark.timeout(400)
+
+def test_hotplug(qemu_instance_per_fcn):
+    qemu_instance_per_fcn['serial2'].sendline('root')
+    cpu_path = '/sys/devices/system/cpu/'
+
+    for i in range(1,8):
```

Comment:
Also, I believe you're only checking cores 1-7.  Any problem with checking core 0 as well?

Reply:
In the link David Kang sent me, it said that CPU 0 is often excluded from hotplugging. Should I include it?

Reply:
Core 0 may be special, so I'm not sure if you can hotplug with it, but why don't you try it once and see?  If it works, include it, otherwise keep the test as is.

Reply:
I was able to turn CPU0 off and verify it's status but when I turned it back on QEMU crashed with a random repeating character on the HPPS serial port.

Reply:
Thanks.  I guess core 0 shouldn't be disabled then.